content/hi/docs/home/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: कुबेरनेट्स प्रलेखन  
+noedit: true
+cid: docsHome
+layout: docsportal_home
+class: gridPage gridPageHome
+linkTitle: "होम"
+main_menu: true
+weight: 10
+hide_feedback: true
+menu:
+  main:
+    title: "प्रलेखन"
+    weight: 20
+    post: >
+      <p>वैचारिक, ट्यूटोरियल और संदर्भ प्रलेखन  के साथ कुबेरनेट्स का उपयोग करना सीखें| आप <a href="/editdocs/" data-auto-burger-exclude data-proofer-ignore>डॉक्स में योगदान करने में</a>! भी मदद कर सकते हैं</p>
+description: >
+  कुबेरनेट्स कंटेनरीकृत अनुप्रयोगों के परिनियोजन, स्केलिंग और प्रबंधन को स्वचालित करने के लिए एक ओपन सोर्स कंटेनर ऑर्केस्ट्रेशन इंजन है| यह ओपन सोर्स प्रोजेक्ट को Cloud Native Computing Foundation द्वारा होस्ट किया गया है।
+overview: >
+  कुबेरनेट्स कंटेनरीकृत अनुप्रयोगों के परिनियोजन, स्केलिंग और प्रबंधन को स्वचालित करने के लिए एक ओपन सोर्स कंटेनर ऑर्केस्ट्रेशन इंजन है। यह ओपन सोर्स प्रोजेक्ट को Cloud Native Computing Foundation द्वारा होस्ट किया गया है|(<a href="https://www.cncf.io/about">CNCF</a>).
+cards:
+- name: concepts
+  title: "मूल बातें समझें"
+  description: "कुबेरनेट्स और इसकी मूलभूत अवधारणाओं के बारे में जानें|"
+  button: "अवधारणाओं को जानें"
+  button_path: "/hi/docs/concepts"
+- name: tutorials
+  title: "कुबेरनेट्स को उपयोग कर ने का प्रयास करें"
+  description: "कुबेरनेट्स में एप्लिकेशन को परिनियोजित करने का तरीका जानने के लिए ट्यूटोरियल का अनुसरण करें|"
+  button: "ट्यूटोरियल देखें"
+  button_path: "/hi/docs/tutorials"
+- name: setup
+  title: "एक कुबेरनेट्स क्लस्टर सेट करें"
+  description: "अपने संसाधनों और जरूरतों के आधार पर कुबेरनेट्स को चालू करें|"
+  button: "कुबेरनेट्स को सेट करें "
+  button_path: "/hi/docs/setup"
+- name: tasks
+  title: "कुबेरनेट्स का उपयोग करना सीखें"
+  description: "सामान्य कार्यों को देखें और चरणों के एक छोटे क्रम का उपयोग करके उन्हें करें|"
+  button: "कार्य देखें"
+  button_path: "/hi/docs/tasks"
```

Comment:
To make this work, you'll need to localize `/docs/tasks` (which is OK, but the link target needs to exist).

Reply:
we'll be localization that file as well, right, in the near future !! So, for the time-being, should we remove this or keep that as it is (again, taking ref. from `Korean `localization), but i do get your point here 🤔

Reply:
```suggestion
  button_path: "/docs/tasks"
```
_for now_